specs/commandline/sign.md

```diff
@@ -0,0 +1,65 @@
+# notation sign
+## Description
+Use `notation sign` to sign artifacts.
+
+Signs a container artifact that is stored in a registry. Upon successful signing, the generated signature is pushed to the registry and the digest of the container image is returned.
+## Outline
+```
+Sign artifacts
+
+Usage:
+  notation sign <reference> [flags]
+
+Flags:
+      --cert-file string        Location of file containing signing(leaf) certificate and certificate chain. Use this flag with flag '--key-file' together.
+  -e, --expiry duration         Optional expiry that provides a “best by use” time for the artifact. The duration is specified in seconds, minutes or hours.
+  -h, --help                    Help for sign
+  -k, --key string              Signing key name, for a key previously added to notation's key list.
+      --key-file string         Signing key file. Use this flag with flag '--cert-file' together.
+  -p, --password string         Password for registry operations (default to $NOTATION_PASSWORD if not specified)
+  -c, --pluginConfig string     Optional list of comma-separated {key}={value} pairs that are passed as is to a plugin, if the key (--key) is associated with a signing plugin, refer plugin documentation to set appropriate values
+  -u, --username string         Username for registry operations (default to $NOTATION_USERNAME if not specified)
+
+Global Flags:
+      --plain-http   Registry access via plain HTTP
+```
+## Usage
+### Sign a container image
+```
+# Add a key which uses a local private key and certificate, and make it a default signing key
+notation key add --name <key name> <key path> <cert path> --default
+
+# [Optional] Change a default signing key
+notation key update <key name> --default
+
+# Sign a container image using the default signing key
+notation sign https://<registry>/<repository>:<tag>
+```
+### Sign a container image using a remote key
+```
+# Prerequisites: 
+# - A compliant signing plugin is installed in notation. See notation plugin documentation for more details (https://github.com/notaryproject/notaryproject/blob/main/specs/plugin-extensibility.md).
```

Comment:
```suggestion
# - A compliant signing plugin is installed in notation. See notation [plugin documentation](https://github.com/notaryproject/notaryproject/blob/main/specs/plugin-extensibility.md) for more details.
```

Reply:
This link is under a block, so it won't work using format `[](https://)`